trie/statistics/trieStatisticsCollector.go

```diff
@@ -55,6 +55,11 @@ func (tsc *trieStatisticsCollector) Print() {
 	)
 }
 
+// GetNumNodes returns the number of nodes
```

Comment:
I know this is a problem not related to your PR but we need to fix this:
L36 can easily panic if I send a nil object. Either nil check or change the definition of the Add function to pass the argument by value

Reply:
pushed, passing by value now